### PR TITLE
feat(excel2json-lists): enable to add comments to list nodes (RDU-18)

### PR DESCRIPTION
**Pull request by Nora-Olivia-Ammann**

### CODE REVIEW

File: docs/file-formats/excel2json.md [hunk 1]
```diff
@@ -375,6 +379,11 @@ The Excel sheets must have the following structure:
       For example:
           - `list1:node1:node1.1` for the node `node1.1` in `list1`
           - `list2:node1:node1.1` for the node `node1.1` in `list2`
+- `<language>_comments`: These are optional but recommended at least for the list itself (the second row of the Excel).
+  If they are not provided for the list, the labels will be entered in the comments section.
+  Nodes will not have comments if not explicitly provided.
+  Same with the nodes, all used languages have to be entered.
```
Comment (jnussbaum) [inline docs/file-formats/excel2json.md:385]: What means "same with the nodes"? Is there a connection between the first part of this line (before the comma), and the 2nd part of this line (after the comma)?

File: docs/assets/data_model_templates/lists/new_lists.xlsx
Comment (jnussbaum) [inline docs/assets/data_model_templates/lists/new_lists.xlsx:1]: This file should be mentioned in https://github.com/dasch-swiss/dsp-tools/blob/2310abc2e1b06af8fec59bcee4fd1a889830e552/docs/file-formats/excel2json.md#L256

**Review by jnussbaum — APPROVED** [review]
LGTM

File: docs/assets/data_model_templates/lists/new_lists.xlsx
Comment (Nora-Olivia-Ammann) [reply docs/assets/data_model_templates/lists/new_lists.xlsx:1]: This is for the old lists. There is a link to the new template at the end of the file after the new lists section is explained

**Pull request merged**